mne/viz/evoked.py

```diff
@@ -1191,3 +1191,349 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     # show and return it
     plt_show(show)
     return fig
+
+
+def _ci(arr, ci):
+    """Calculate the `ci`% parametric confidence interval for `arr`.
+    Aux function for plot_compare_evokeds."""
+    from scipy import stats
+    mean, sigma = arr.mean(0), stats.sem(arr, 0)
+    # This is highly convoluted to support 17th century Scipy
+    # XXX Fix when Python 2.6 support is dropped!
+    return np.asarray([stats.t.interval(ci, arr.shape[0],
+                       loc=mean_, scale=sigma_)
+                       for mean_, sigma_ in zip(mean, sigma)]).T
+
+
+def _setup_styles(conditions, style_dict, style, default):
+    """Aux function for plot_compare_evokeds to set linestyles and colors"""
+    condition_warning = "Condition {} could not be mapped to a " + style
+    style_warning = ". Using the default of {}".format(default)
```

Comment:
style_warning = ". Using the default of {0}".format(default)

to make travis happy on Py 2.6